### DISCUSSION TITLE
It is possible to create own relational data tables( not cms's) for specific module

### DISCUSSION THREAD

**Discussion by ricocsharp** [OrchardCMS/OrchardCore#13832] [Q&A]
Hi, It is posible to create relational tables in orchard core database for specific module?  How to create them using migrations?

Bests
Javier

**Comment by hishamco** [comment]
This is doable, you could create custom tables inside the migration using `SchemaBuilder` property

**Comment by ns8482e** [comment]
All you need is to have access to database connection string to do anything with orchard database. You can use db accessor that OC provides example here -->https://docs.orchardcore.net/en/latest/docs/reference/core/Data/#example

**Reply by hishamco** [reply]
Maybe I didn't see this example before. Also, `SchemaBuilder` will allow you to create the schema for your custom table if you need so

**Comment by ricocsharp** [comment]
OK thanks a lot to everyone I will check this twoo ways for doing this
